app/services/strategy_submission_service.py

```
@@ -0,0 +1,798 @@
+        submission = result.scalar_one_or_none()
+        if not submission:
+            raise ValueError("Strategy submission not found")
+        return submission
+
+    async def _promote_submission_to_marketplace(
+        self, submission: StrategySubmission, db: AsyncSession
+    ) -> None:
+        user_uuid = uuid.UUID(submission.user_id)
+        user = await db.get(User, user_uuid)
+        if not user:
+            raise ValueError("Submission owner no longer exists")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)